docs/src/config.ts

```diff
@@ -20,6 +20,7 @@ export const SIDEBAR = {
     { text: 'Data Fetching', link: 'guides/data-fetching' },
     { text: 'Markdown', link: 'guides/markdown-content' },
     { text: 'Supported Imports', link: 'guides/imports' },
+    { text: 'Aliases', link: 'guides/aliases' },
```

Comment:
I dont think this should be its own page, having reviewed it I believe this is a better fit to help as an appendix to the `references/configuration-reference` file, places under the content that is already there.

Reply:
We'd pretty much be copy/pasting it from the [Snowpack documentation](https://www.snowpack.dev/reference/configuration#alias) and there's already a link for whole thing there. If we add this we might as well add everything regarding `snowpack.config`, that wouldn't be very nice because in my opinion Astro and Snowpack are advertised as two separate things.

Reply:
Given how common this question is I kind of like it as a top-level page, at least for now. +1 for keeping as-is for now (top-level page), and we can do a big docs review before a v1 release and move around if needed then.